src/App.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import logo from './logo.svg';
+/* import React from 'react'; */
 import './App.css';
+import It1 from './components/Main/It1'
+import Greetings from './components/Main/Greetings';
+import Random from './components/Main/Random';
+import BoxColor from './components/Main/BoxColor';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+     <div><h1 style={{textAlign:"left", marginLeft:"5px"}}>IdCard</h1></div>     
+     <It1 />
+     <div><h1 style={{textAlign:"left", marginLeft:"5px"}}>Greetings</h1></div>
+     <Greetings lang="de"><p>Ludwig</p></Greetings>
```

Comment:
Ten cuidado no te muestra el children de este por que es una tag 👀 
```suggestion
     <Greetings lang="de">Ludwig</Greetings>
```